concordium-node/src/rpc.rs

```diff
@@ -429,11 +429,20 @@ impl P2p for RpcServerImpl {
         Ok(Response::new(match self.consensus {
             Some(ref consensus) => {
                 let consensus_baking_committee_status = consensus.in_baking_committee();
+                let consensus_baker_running =
+                    if let ConsensusIsInBakingCommitteeResponse::ShutDown =
+                        consensus_baking_committee_status
+                    {
+                        false
+                    } else {
+                        consensus.is_baking()
+                    };
+
                 NodeInfoResponse {
                     node_id,
                     current_localtime,
                     peer_type,
-                    consensus_baker_running: consensus.is_baking(),
+                    consensus_baker_running,
                     consensus_running: true,
```

Comment:
If the consensus is shutdown, then this probably should be `false`.

Reply:
That was indeed the entire point of the bug, that this reports true when it is shut down.